ipynb/tests/Frequency_Invariance_Test.ipynb

```diff
@@ -0,0 +1,799 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f470b406e50>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_signal_against_freq('load_avg')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Examine trace from workload execution"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plot task residency and `sched_util` and `sched_load` for the workload task, along with the expected mean value for util_avg. Note that assuming the system was under little load, so that the task was RUNNING whenever it was RUNNABLE, `load_avg` and `util_avg` should be the same. \n",
+    "\n",
+    "Call `examine_experiment` with different experiment indexes to get plots for runs at different frequencies."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "signals = ['util_avg', 'load_avg']\n",
+    "def examine_experiment(idx):\n",
+    "    experiment = t.executor.experiments[idx]\n",
```

Comment:
OK, you think it should be a method (rather than raw attribute)? In that case it should be `get_experiment` not `experiment`.

But actually I prefer a raw attribute to a method, because then you can easily iterate over it.